platform/index.md

```diff
@@ -22,4 +74,12 @@ redirects:
 
 [![ServicePulse article index](/menu/servicepulse-logo.png)](/servicepulse)
 
-[![ServiceControl article index](/menu/servicecontrol-logo.png)](/servicecontrol)
\ No newline at end of file
+[![ServiceControl article index](/menu/servicecontrol-logo.png)](/servicecontrol)
+
+To see demos showing how those tools are used and to understand the principles behind the platform even better, watch the videos:
+
+* [Particular Service Platform Overview](http://player.vimeo.com/video/99322069?autoplay=1), 
```

Comment:
Seems like we can only navigate to this video from particular.net page. I'll remove it.